change/@uifabric-charting-2020-03-16-12-29-29-user-v-jasha-verticalStackedBarChart.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "minor",
+  "comment": "removed test file",
```

Comment:
Please provide a detailed description and update your email (it's currently not defined) - these go out in release notes to all our users.

Reply:
description and email added.